src/mots/cli.py

```diff
@@ -68,7 +68,8 @@ def clean(args: argparse.Namespace) -> None:
         messages = (
             "Either set it in your environment (MOTS_BUGZILLA_API KEY) or add it to "
             "your settings file by running: "
-            "`mots settings write BUGZILLA_API_KEY <your API key>`.",
+            "`mots settings write BUGZILLA_API_KEY`.",
+            "You will be prompted to enter the API key after running this command."
```

Comment:
We do still support `mots settings write <KEY> <VALUE>`, but we don't want to recommend it for secure values IIUC?

Reply:
Yes that's correct, so it doesn't end up in bash history.